linux_os/guide/services/http/disabling_httpd/package_httpd_removed/rule.yml

```diff
@@ -4,7 +4,12 @@ prodtype: rhel7,rhel8,rhel9,sle15
 
 title: 'Uninstall httpd Package'
 
-description: '{{{ describe_package_remove(package="httpd") }}}'
+description: |-
```

Comment:
I see that you change the description here but you don't do it in most of other rules touched by this PR. Any reason for that?

Reply:
I was just being lazy. :D 

I've updated some, I think we need to be more consistent about using `describe_package_remove` and fix up `ocil_entry` but I feel they're out of the scope of this PR.  Once this merges, I can open a new one if desired.